src/pdf.js

```diff
@@ -19,6 +19,8 @@
 /** @typedef {import("./display/api").PDFPageProxy} PDFPageProxy */
 /** @typedef {import("./display/api").RenderTask} RenderTask */
 /** @typedef {import("./display/display_utils").PageViewport} PageViewport */
+// eslint-disable-next-line max-len
+/** @typedef {import("./display/text_layer.js").TextLayerRenderTask} TextLayerRenderTask */
```

Comment:
Looking at the lines above this should not have the `.js` extension in the import statement.